odl/util/utility.py

```diff
@@ -34,131 +34,118 @@
 TYPE_MAP_C2R.update({k: k for k in TYPE_MAP_R2C.keys()})
 
 
-def indent_rows(string, indent=4):
-    """Return ``string`` indented by ``indent`` spaces."""
-    return '\n'.join((' ' * indent) + row for row in string.split('\n'))
+def indent(string, indent_str='    '):
+    """Return a copy of ``string`` indented by ``indent_str``."""
```

Comment:
Might as well add a doctest for such a widely used function

Reply:
Done